DxDispatch/src/model/ImageReaderWriter.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+enum class ImageTensorLayout
+{
+    NCHW,
+    NHWC,
+};
+
+enum class ImageTensorChannelOrder
+{
+    Unspecified,
+    RGB,
+    RGBA,
+    BGR,
+    Grayscale,
+};
+
+struct ImageTensorInfo
+{
+    DML_TENSOR_DATA_TYPE dataType;
+    uint32_t channels;
+    uint32_t height;
+    uint32_t width;
+    uint64_t sizeInBytes;
+    ImageTensorLayout layout;
+    ImageTensorChannelOrder channelOrder;
+};
+
+std::vector<std::byte> ReadTensorFromImage(
+    std::filesystem::path srcPath,
```

Comment:
```suggestion
    std::filesystem::path const& srcPath,
```
I presume it's readonly.